posthog-node/CHANGELOG.md

```diff
@@ -27,6 +27,7 @@
 ## Breaking changes
 
 1. feat: migrate to native fetch, Node 20+ required
+2. PostHog Node now compresses messages with GZip before sending them to our servers when the runtime supports compression. Network traffic interceptors and test assertions on payloads must handle GZip decompression to inspect the data
```

Comment:
Alternatively, one can set `{ disableCompression: true }` in the client instantiated in the tests. (That's how we're doing it.)